### PR TITLE
Feature/23 create backend seeder

### PR DESCRIPTION
**Pull request by wayangalihpratama**
- #23 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207597067843317

### CODE REVIEW

File: backend/tests/seeder/test_conversation_demo_seeder.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+def conversation_data():
+    json_path = "./seeder/conversation_demo.json"
+    with open(json_path, "r") as f:
+        data = json.load(f)
+    return data
+
+
+def test_create_client(session: Session):
+    create_client(session=session)
+    result = session.exec(text("SELECT COUNT(*) FROM \"client\""))
```
Comment (dedenbangkit) [inline backend/tests/seeder/test_conversation_demo_seeder.py:22]: please use sqlmodel command for consistency

File: backend/tests/seeder/test_conversation_demo_seeder.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    user_id = last_user.id
+    client_id = create_client(session=session)
+    assert client_id is not None
+
+    seed_chat_data(session=session, user_id=user_id, client_id=client_id)
+
+    # Verify chat sessions after seeding
+    chat_sessions = session.exec(
+        text("SELECT * FROM chat_session ORDER BY id DESC")
```
Comment (dedenbangkit) [inline backend/tests/seeder/test_conversation_demo_seeder.py:38]: please use sqlmodel command for consistency

File: backend/tests/seeder/test_conversation_demo_seeder.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    ).fetchall()
+
+    # Find the chat session ID for the created session
+    chat_session_id = chat_sessions[0].id if chat_sessions else None
+    assert chat_session_id is not None
+
+    # Verify that chat messages are seeded correctly
+    chat_messages = session.exec(text(
+        "SELECT * FROM chat WHERE chat_session_id = :chat_session_id"
```
Comment (dedenbangkit) [inline backend/tests/seeder/test_conversation_demo_seeder.py:47]: please use sqlmodel command for consistency

File: backend/tests/seeder/test_user_seeder.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        "123456789",  # Phone number
+        "y",           # User has properties
+        "John Doe",    # Name
+        "john.doe@example.com",  # Email
+    ]
+    monkeypatch.setattr('builtins.input', lambda _: user_input.pop(0))
+    interactive_seeder(session=session)
+    phone_number = "123456789"
+    result = session.exec(text(
+        "SELECT * FROM \"user\" WHERE phone_number = :phone_number"
```
Comment (dedenbangkit) [inline backend/tests/seeder/test_user_seeder.py:27]: please use sqlmodel command for consistency

File: backend/tests/seeder/test_user_seeder.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    ).bindparams(phone_number=phone_number)).fetchall()
+    assert len(result) == 1
+    user_id = result[0].id
+    result_up = session.exec(text(
+        "SELECT COUNT(*) FROM \"user_properties\" WHERE user_id = :user_id"
```
Comment (dedenbangkit) [inline backend/tests/seeder/test_user_seeder.py:32]: please use sqlmodel command for consistency

**Review by dedenbangkit — CHANGES_REQUESTED** [review]

File: backend/tests/seeder/test_conversation_demo_seeder.py (hunk 2 above)
Comment (wayangalihpratama) [reply backend/tests/seeder/test_conversation_demo_seeder.py]: Updated mas @dedenbangkit

File: backend/tests/seeder/test_conversation_demo_seeder.py (hunk 3 above)
Comment (wayangalihpratama) [reply backend/tests/seeder/test_conversation_demo_seeder.py]: Updated mas @dedenbangkit

File: backend/tests/seeder/test_user_seeder.py (hunk 4 above)
Comment (wayangalihpratama) [reply backend/tests/seeder/test_user_seeder.py]: Updated mas @dedenbangkit

File: backend/tests/seeder/test_user_seeder.py (hunk 5 above)
Comment (wayangalihpratama) [reply backend/tests/seeder/test_user_seeder.py]: Updated mas @dedenbangkit

**Review by dedenbangkit — APPROVED** [review]

**Review by kjkoster — APPROVED** [review]

**Pull request merged**